go/pkg/services/logcollector/server.go

```diff
@@ -611,16 +606,12 @@ func (s *Server) startLogStreaming(ctx context.Context,
 	runUID,
 	podName,
 	projectName string,
-	startedStreamingGoroutine chan bool,
-	cancelCtxFunc context.CancelFunc) {
+	startedStreamingGoroutine chan bool) {
 
 	// in case of a panic, remove this goroutine from the current state, so the
 	// monitoring loop will start logging again for this runUID.
 	defer func() {
 
-		// cancel all other goroutines spawned from this one
-		defer cancelCtxFunc()
```

Comment:
I think there was a reason for this cancellation but I cant find other goroutines that are spawned from this context in the current flow.. Maybe something changed along the way.